app/helpers/application_helper.rb

```diff
@@ -71,31 +71,29 @@ def alert_container_class_for_flash_type type
     base_class
   end
 
-  def filter_badge(**args)
-    label = args[:label]
-    model = args[:model]
-    filter_by = args[:filter_by]
-
+  def filter_badge(label: nil, model: nil, filter_by: nil, color: nil, title: nil)
```

Comment:
(IMO kwargs are more explicit for documenting options you can pass to a method)

Reply:
feel free to make it better!